runtime/common/src/parachains.rs

```diff
@@ -1248,7 +1248,7 @@ impl<T: Trait> Module<T> {
 					Error::<T>::DisallowedCodeUpgrade,
 				);
 				ensure!(
-					schedule.max_code_size >= new_code.len() as u32,
+					schedule.max_code_size >= new_code.0.len() as u32,
```

Comment:
Probably it's no longer needed to keep the inner field of `ValidationCode` and other newtypes public? They already have `From<Vec>` implementations and they could also have  `AsRef<[u8]>`. Then the code here would become `new_code.len()`.

Reply:
That would require `Deref<[u8]>` which is not conventionally used for anything but smart pointers. But yeah, then we'd use `.as_ref().len()` which seems fine